test/unittests/test_http.py

```diff
@@ -13,6 +13,7 @@
              'after_cursor': 'some_cursor'}
```

Comment:
@prijendev Did we write test cases for different scenarios of `request_timeout` variable?

Reply:
Yes, We write test cases for different scenarios of request_timeout variable in test_request_timeout.py file.